### PR TITLE
when editing an editable field, show original content for reference

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: src/field/index.jsx [hunk 1]
```diff
@@ -38,12 +38,15 @@ class Field extends Component {
...
-            />
+            ? <Fragment>
+              <ReactMarkdown>{ content }</ReactMarkdown>
+              <TextArea
+                label=""
+                name={name}
+                value={this.state ? this.state.content : content}
+                onChange={this.onChange}
+              />
+            </Fragment>
```
Comment (lennym) [inline src/field/index.jsx]: You can get rid of the `Fragment` and the ternary by pulling the `<ReactMarkdown>` bit up outside the conditional.

Comment (joefitter) [reply src/field/index.jsx]: you certainly can

Comment (lennym) [reply src/field/index.jsx]: You should consider it.

Comment (joefitter) [reply src/field/index.jsx]: I will

**Pull request merged**